build/linux/installer/conf/kube.conf

```diff
@@ -4,6 +4,8 @@
     root_dir /var/opt/microsoft/docker-cimprov/state
    </system>
 
+   #@include windows_rs.conf
```

Comment:
can you please validate this change with CPU limit 2 or more cores on the replicaset pod?  In the RS pod, If the CPU limit more than 1 core, fluentd plugins will be running on multiple fluentd processes.